portfolio/src/main/webapp/index.html

```diff
@@ -1,23 +1,219 @@
-      <button onclick="addRandomFact()">Fact</button>
-      <div id="fact-container"></div>
-    </div>
-  </body>
+    
+
+<body id="myPage" data-spy="scroll" data-target=".navbar" data-offset="50">
+
+<nav class="navbar navbar-default navbar-fixed-top">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>                        
+      </button>
+      <a class="navbar-brand" href="#myPage">MICHELLE KROLL'S PORTFOLIO</a>
+    </div>
+    <div class="collapse navbar-collapse" id="myNavbar">
+      <ul class="nav navbar-nav navbar-right">
+        <li><a href="#myPage">HOME</a></li>
+        <li><a href="#about">ABOUT ME</a></li>
+        <li><a href="#projects">PROJECTS</a></li>
+        <li><a href="#contact">CONTACT</a></li>
+        <li><a href="#"><span class="glyphicon glyphicon-search"></span></a></li>
+      </ul>
+    </div>
+  </div>
+</nav>
+
+<div id="myCarousel" class="carousel slide" data-ride="carousel">
+    <!-- Indicators -->
+    <ol class="carousel-indicators">
+      <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+      <li data-target="#myCarousel" data-slide-to="1"></li>
+      <li data-target="#myCarousel" data-slide-to="2"></li>
+    </ol>
+
+    <!-- Wrapper for slides -->
+    <div class="carousel-inner" role="listbox">
+      <div class="item active">
+        <img src="/images/MichellePhoto.JPG" alt="Michelle" width="500" height="700">
+        <div class="carousel-caption">
+          <h3>Hey, I'm Michelle Kroll.</h3>
+          <p>It's nice to meet you.</p>
+        </div>      
+      </div>
+    </div>
+
+    <!-- Left and right controls -->
+    <a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
+      <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+      <span class="sr-only">Previous</span>
+    </a>
+    <a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
+      <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+      <span class="sr-only">Next</span>
+    </a>
+</div>
+
+<!-- Container (The About Me Section) -->
+<div id="about" class="container text-center">
+  <h3>ABOUT ME</h3>
+  <p><em>Welcome to my portfolio!</em></p>
+  <p>My name is Michelle Kroll and I am a second-year student at UC Berkeley studying Computer Science. I am a goal-driven and motivated student and am interested in pursuing a career in software engineering.</p>
+  <br>
+  <div class="row">
+    <div class="col-sm-4">
+      <p class="text-center"><strong>GitHub</strong></p><br>
+      <p><a href="https://github.com/michellekroll">
+    <img src="/images/GitHub Logo.png" class="img-circle person" alt="Random Name" width="255" height="255">
+    </a></p>
```

Comment:
Generally speaking, it's easier to read with the opening and ending tags on the same column, so in this case 
```
<p>
  <a href="https://github.com/michellekroll">
    <img src="/images/GitHub Logo.png" class="img-circle person" alt="Random Name" width="255" height="255">
  </a>
</p>
```
This makes it easier to read and see what is nested.
Here and throughout the file